.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -74,7 +74,7 @@ jobs:
         run: docker-compose pull test || true
 
       - name: Run codebase test
-        run: docker-compose run test bundle exec rspec spec/codebase/codebase_spec.rb --format progress
+        run: docker-compose run test -e DISABLE_SPRING=1 bundle exec rspec spec/codebase/codebase_spec.rb --format progress
```

Comment:
Hi @sanG-github we are facing an error due to this changes in our new project, could you double check that, we removed the `-e DISABLE_SPRING=1` now in our new project, thank you

<img width="1002" alt="image" src="https://github.com/nimblehq/rails-templates/assets/11751745/f1dffe21-4342-448e-b87a-eb2ac88fd31d">